implementation/src/main/java/io/smallrye/faulttolerance/CompositeObservableCommand.java

```diff
@@ -15,40 +15,40 @@
  */
 package io.smallrye.faulttolerance;
 
+import java.lang.reflect.Field;
+import java.security.PrivilegedActionException;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Supplier;
+
+import org.eclipse.microprofile.metrics.Counter;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.MetricType;
+
 import com.netflix.hystrix.HystrixCommandKey;
 import com.netflix.hystrix.HystrixCommandProperties;
 import com.netflix.hystrix.HystrixObservableCommand;
 import com.netflix.hystrix.HystrixThreadPoolKey;
 import com.netflix.hystrix.HystrixThreadPoolProperties;
+
 import io.smallrye.faulttolerance.config.FaultToleranceOperation;
 import io.smallrye.faulttolerance.metrics.MetricNames;
 import io.smallrye.faulttolerance.metrics.MetricsCollectorFactory;
-import org.eclipse.microprofile.metrics.Counter;
-import org.eclipse.microprofile.metrics.MetricRegistry;
-import org.eclipse.microprofile.metrics.MetricType;
 import rx.Observable;
 
-import java.lang.reflect.Field;
-import java.security.PrivilegedActionException;
-import java.util.concurrent.Callable;
-import java.util.concurrent.CompletionStage;
-import java.util.function.Supplier;
-
 /**
  *
  * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
- * <br>
- * Date: 2/7/19
  */
 public class CompositeObservableCommand extends HystrixObservableCommand {
 
     public static HystrixObservableCommand<?> create(Callable<? extends CompletionStage<?>> callable,
-                                                     FaultToleranceOperation operation,
-                                                     RetryContext retryContext,
-                                                     ExecutionContextWithInvocationContext ctx,
-                                                     MetricRegistry registry,
-                                                     boolean timeoutEnabled,
-                                                     Supplier<Object> fallback) {
+            FaultToleranceOperation operation,
```

Comment:
:( former was nicer. Maybe moving callable to a next line would make it a bit beter.

Reply:
Yea, it's based on Eclipse :-(

I'm personally ambivalent on indenting formal parameters. I usually do this:

```java
public static HystrixObservableCommand<?> create(Callable<? extends CompletionStage<?>> callable,
        FaultToleranceOperation operation, RetryContext retryContext, ExecutionContextWithInvocationContext ctx,
        MetricRegistry registry, boolean timeoutEnabled, Supplier<Object> fallback) {
    ...
}
```

But I just couldn't go through it all and fix everything I didn't like.